tests/utils/encoding/tests.py

```diff
@@ -227,6 +227,12 @@ def test_shorten_tuple():
     #     assert result[-1] == '(450 more elements)'
 
 
+def test_shorten_dict():
+    result = shorten({k: v for k in range(500) for v in range(500)}, dict_length=50)
```

Comment:
```suggestion
    result = shorten({k: k for k in range(500)}, dict_length=50)
```
The double loop produces a dict where all values are `499`

Reply:
Whoops, I don't know what I was thinking. To accomplish what I meant to accomplish, I'm actually going to do this:

```
{k: v for k, v in enumerate(range(500))}
```

Reply:
@beniwohli pointed out that my version is actually the same as your version, but way less efficient. I blame newborn sleep deprivation! Fixed in https://github.com/elastic/apm-agent-python/pull/638/commits/8827348f8d37302c1c1bc6b5a7104dbe78660b67